docs/sources/reference/commandline/cli.md

```diff
@@ -847,6 +848,11 @@ you refer to it on the command line.
 > children) for security reasons, and to ensure repeatable builds on remote
 > Docker hosts. This is also the reason why `ADD ../file` will not work.
 
+When `docker build` is run with the `--cgroup-parent` option the containers used
+in the build will be run with the [corresponding `docker run`
+flag](/reference/run/#specifying-custom-cgroups). 
```

Comment:
Not entirely happy with this sentence; I had some trouble understanding what was meant here, and almost interpreted *"will be run with the corresponding docker run flag"* as *"any docker run flag can be set here"*, (e.g. `--cgroup-parent=volumes-from=something`).

Perhaps something like; When docker build is run with the `--cgroup-parent`, the containers used in the build are run with the specified cgroup. For more information, see "Specifying custom cgroups"


**note** I just noticed that the `--cgroup-parent` flag for `docker run` [isn't documented](https://github.com/docker/docker/blob/8bd5a95e1e39541dfc5dc635c5c8e7604fe10028/docs/sources/reference/commandline/cli.md#run) in the command-line reference